packages/db/src/scripts/import-bdb.ts

```diff
@@ -319,6 +319,12 @@ function processEntry(entry: string): string {
       .replaceAll(/<(\/?)big>?/g, '<$1strong>')
       // Remove hrefs from hyperlinks
       .replaceAll(/href=".*?"/g, '')
+      // Add ref class so refs can be handled for previews
+      .replaceAll('<a', '<a class="ref"')
```

Comment:
I'm seeing an artifact of the import process. This was in Genesis 1:1 on elohim.
<img width="204" alt="Screen Shot 2024-04-04 at 9 09 15 PM" src="https://github.com/globalbibletools/gbt/assets/11968925/4682e1b9-982d-420a-8923-a4aae28845d9">